src/components/promo.tsx

```diff
@@ -71,6 +71,11 @@ export const Promo = () => {
       link: "https://pris.ly/sidebar-promo/sql-queries-in-orm",
       color: "teal",
     },
+    {
+      text: `Tell us how you like working with Prisma ORM in your projects!`,
```

Comment:
I read this an heard your voice..... anyways.

Do you like working with Prisma ORM in your projects? Let us know! 

Just an idea

Reply:
I like that more, but feels less assertive (?) to make people fill it out even if they don't like the ORM.

This is the monthly NPS survey that we run at the end of the quarter.

How about "Tell us how Prisma ORM is working for you!"?

Alts:
How has Prisma ORM been helping you in your projects? We'd love your feedback!
Share your experience with Prisma ORM in your projects!
Enjoying Prisma ORM? We'd love to hear about your experience!
How has Prisma ORM improved your project development? Let us know!
Tell us how Prisma ORM is working for you!
Your thoughts on Prisma ORM matter—share them with us!
How's Prisma ORM fitting into your workflow? We'd like to know!
Help us improve—share your experience with Prisma ORM!
We value your opinion! How has Prisma ORM been for your projects?
How do you feel about using Prisma ORM? Your feedback is important!

Reply:
Yea a question is just an easier click in my opinion. 

""Tell us how Prisma ORM is working for you!"?" is good!